hs_file_types/models/netcdf.py

```diff
@@ -167,6 +167,7 @@ def get_html_forms(self, dataset_name_form=True, temporal_coverage=True, **kwarg
     def get_update_netcdf_file_html_form(self):
         form_action = "/hsapi/_internal/{}/update-netcdf-file/".format(self.logical_file.id)
         style = "display:none;"
+        self.refresh_from_db()
```

Comment:
I don't get these db refreshes.

Reply:
For the same reason as explained previously (above).